server/database/index.js

```diff
@@ -19,17 +24,46 @@ Group.hasMany(Chore);
 /*----- Vote Associations -----*/
 User.hasMany(Vote, { foreignKey: 'voterId', sourceKey: 'id' });
 Vote.belongsTo(User, { foreignKey: 'voterId', targetKey: 'id' });
-// Vote.belongsTo(AssignedChore)
+// Vote.belongsTo(AssignedChores)
 // AssignedChore.hasMany(Vote);
 
 /*----- Trade Associations -----*/
 Trade.belongsTo(User, { foreignKey: 'originalOwnerId' });
 Trade.belongsTo(User, { foreignKey: 'newOwnerId' });
 User.hasMany(Trade, { foreignKey: 'originalOwnerId' });
 User.hasMany(Trade, { foreignKey: 'newOwnerId' });
-// Trade.belongsTo(AssignedChore)
+// Trade.belongsTo(AssignedChores)
```

Comment:
Singular

Reply:
any reason we want to keep these commented out code?  Fixed in the meantime.